src/config.rs

```diff
@@ -4318,4 +4318,41 @@ mod tests {
             assert_eq!(Module::from(name), module);
         }
     }
+
+    #[test]
+    fn test_compaction_style() {
+        let normal_string_config = r#"
+            compaction-style = "universal"
+        "#;
+        let config: DefaultCfConfig = toml::from_str(normal_string_config).unwrap();
+        assert_eq!(config.compaction_style, DBCompactionStyle::Universal);
+
+        // Test backward compatibility with numeric value
+        let normal_string_config = r#"
+            compaction-style = 1
+        "#;
+        let config: DefaultCfConfig = toml::from_str(normal_string_config).unwrap();
+        assert_eq!(config.compaction_style, DBCompactionStyle::Universal);
+        assert!(
+            toml::to_string(&config)
+                .unwrap()
+                .contains("compaction-style = \"universal\"")
```

Comment:
it will deserialize to a string, and older versions can't recognize that. @tabokie

Reply:
Ok, then let's serialize to integers. @yuqi1129

Reply:
I'm not sure if it's really necessary, just to notice you discuss with PM first.

Reply:
Storing integers is better. I don't want to break anything even if rolling back major version isn't mandatory.

Reply:
If we just use Integers, this PR seems to be redundant. After all if we revert to integer, this issue only make one change: configuration values can accept both numeric or string value. Under this circumstance, All we need to do is just to change the user doc.

Reply:
No, serializing to integer doesn't mean we don't accept strings. It's just keeping a copy of configuration as integer, in `last_tikv.toml`. This way, if user didn't specify these configs in `tikv.toml`, it can rollback without problem.

See https://github.com/tikv/tikv/blob/5552758d277b0ab2761deb88c9a82525ecac8980/src/config.rs#L2918